assets/airports/kpvd.json

```diff
@@ -0,0 +1,6501 @@
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["16", "34"],
+            "end":  [
+                ["N41d43m53.9542",  "W071d25m55.8304", "53.7ft"],
+                ["N41d43m06.8846",  "W071d25m05.9925", "32.8ft"]
+            ],
+            "ils": [false, true]
+        }
+    ],
+    "stars": {
+        "JORDN2": {
+            "icao": "JORDN2",
+            "name": "Jordan Two",
+            "entryPoints": {
+                "HTO": ["HTO"]
+            },
+            "body": ["JORDN", "MINNK", "#015"],
+            "draw": [
+                ["HTO*", "JORDN", "MINNK"]
+            ]
+        },
+        "WIPOR3": {
+            "icao": "WIPOR3",
+            "name": "Wipor Three",
+            "entryPoints": {
+                "ALB": ["ALB", "SHIGY", "PONEE", "NELIE"],
+                "HNK": ["HNK", "HINGZ", "PONEE", "NELIE"],
+                "HINGZ": ["HINGZ", "PONEE", "NELIE"]
+            },
+            "body": ["WIPOR", "ORW", "SAUGY"],
+            "rwy": {
+                "05": ["LAFAY", "#029"],
+                "34": ["LAFAY", "#029"],
+                "16": ["WAWIK", "SEABE", "NOSUH", "#047"],
+                "23": ["WAWIK", "SEABE", "NOSUH", "#047"]
+            },
+            "draw": [
+                ["ALB*", "SHIGY", "PONEE"],
+                ["HNK*", "HINGZ*", "PONEE"],
+                ["PONEE", "NELIE", "WIPOR", "ORW", "SAUGY"],
+                ["SAUGY", "LAFAY"],
+                ["SAUGY", "WAWIK", "SEABE", "NOSUH"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "KPVD",
+            "destination": "",
+            "category": "departure",
+            "route": "KPVD..BAF",
+            "method": "random",
+            "altitude": "",
+            "speed": "",
+            "rate": 2,
+            "airlines": [
+                ["aal",   7],
+                ["cfg",   1],
+                ["dal",   4],
+                ["jbu",   2],
+                ["swa",   8],
+                ["ual",   4]
+            ]
+        },
+        {
+            "origin": "KPVD",
+            "destination": "",
+            "category": "departure",
+            "route": "KPVD..JUMPR",
+            "method": "random",
+            "altitude": "",
+            "speed": "",
+            "rate": 2,
+            "airlines": [
+                ["aal",   7],
+                ["cfg",   1],
+                ["dal",   4],
+                ["jbu",   2],
+                ["swa",   8],
+                ["ual",   4]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KPVD",
+            "category": "arrival",
+            "route": "HTO.JORDN2.KPVD",
+            "method": "random",
+            "altitude": [10000, 12000],
+            "speed": 270,
+            "rate": 1.5,
+            "airlines": [
+                ["aal",   7],
+                ["cfg",   1],
```

Comment:
Seems unlikely condor flies here?

Reply:
@erikquinn they terminated the route recently I think